### ISSUE TITLE
Joining a multiplayer game with expansions when you don't have these

### ISSUE THREAD

**Issue by rbozan** [OpenRCT2/OpenRCT2#2246]
If you join a multiplayer game where the host has the Wacky Worlds expansion, and you're trying to join it when you don't have any expansions, your game will crash and the console will pretty much spam what files you don't have.
```java
ERROR[..\src\object_list.c:554 (object_read_and_load_entries)]: failed to load entry: DIAMONDR
ERROR[..\src\object.c:168 (object_load)]: object not installed
ERROR[..\src\object_list.c:554 (object_read_and_load_entries)]: failed to load entry: HIPPORID
ERROR[..\src\object.c:168 (object_load)]: object not installed
ERROR[..\src\object_list.c:554 (object_read_and_load_entries)]: failed to load entry: MINELIFT
ERROR[..\src\object.c:168 (object_load)]: object not installed
ERROR[..\src\object_list.c:554 (object_read_and_load_entries)]: failed to load entry: OSTRICH
ERROR[..\src\object.c:168 (object_load)]: object not installed
ERROR[..\src\object_list.c:554 (object_read_and_load_entries)]: failed to load entry: RHINORID
ERROR[..\src\object.c:168 (object_load)]: object not installed
```
(... and this continues for the long list of Wacky World objects)

**Comment by rbozan** [comment]
Not sure if this is relevant, but the console ends with this line:
```ERROR[..\src\object_list.c:554 (object_read_and_load_entries)]: failed to load entry: STEX54```
It doesn't say "object not installed" afterwards

**Comment by Gymnasiast** [comment]
That is the scenario description for the WW Victoria Falls scenario.

**Comment by IntelOrca** [comment]
Can be resolved by resolving #1399 so that it caters for Multiplayer.

**Comment by rbozan** [comment]
fixed in 67c807f81f1298f0f144e0f2aa1d0ad0a9b2e4d7
